modules/identity-provider-add.adoc

```diff
@@ -37,15 +48,50 @@ $ oc apply -f </path/to/CR>
 If a CR does not exist, `oc apply` creates a new CR and might trigger the following warning: `Warning: oc apply should be used on resources created by either oc create --save-config or oc apply`. In this case you can safely ignore this warning.
 ====
 
+ifndef::no-username-password-login[]
 . Log in to the cluster as a user from your identity provider, entering the
 password when prompted.
 +
 ----
 $ oc login -u <username>
 ----
+endif::no-username-password-login[]
+
+ifdef::no-username-password-login[]
```

Comment:
unnecessary if-block end?

Reply:
@stlaz No, line 51 is an `ifndef` (if `no-username-password-login` isn't defined) and line 60 is an `ifdef` (if it is defined).

So I was having the above step "Log in to the cluster..." show for IDPs that aren't Google, OIDC, or GitHub. Like it shows for LDAP: https://bz-1795264--ocpdocs.netlify.com/openshift-enterprise/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#add-identity-provider_configuring-ldap-identity-provider

But doesn't show for GitHub, for example here: https://bz-1795264--ocpdocs.netlify.com/openshift-enterprise/latest/authentication/identity_providers/configuring-github-identity-provider.html#add-identity-provider_configuring-github-identity-provider

Reply:
ah, sorry, I missed the `n` in the first condition